viite-UI/src/view/NodeForm.js

```diff
@@ -0,0 +1,101 @@
+(function (root) {
+  root.NodeForm = function (selectedNode) {
+    var NodeType = LinkValues.NodeType;
+    var formCommon = new FormCommon('node-');
+
+    var formButtons = function () {
+      return '<div class="form form-controls">' +
+        '<button class="save btn btn-edit-node-save" disabled>Tallenna</button>' +
+        '<button class="cancel btn btn-edit-node-cancel">Peruuta</button>' +
+        '</div>';
+    };
+
+    var getNodeType = function (nodeValue) {
+      var nodeType = _.find(NodeType, function (type) {
+        return type.value == nodeValue;
+      });
+      return _.isUndefined(nodeType) ? NodeType.UnkownNodeType : nodeType;
+    };
+
+    var staticField = function (labelText, dataField) {
+      var field;
+      field = '<div class="form-group-node-static-metadata">' +
+        '<p class="form-control-static asset-log-info-metadata">' +
+        '<label>' + labelText + '</label>' +
+        dataField +
+        '</p>' +
+        '</div>';
+      return field;
+    };
+
+    var inputField = function (labelText, id, placeholder, value, maxLength) {
+      var lengthLimit = '';
+      if (maxLength)
+        lengthLimit = 'maxlength="' + maxLength + '"';
+      return '<div class="form-group-node-input-metadata">' +
+        '<p class="form-control-static asset-node-data">' +
+        '<label>' + labelText + '</label>' +
+        '<input type="text" class="form-control-static asset-input-node-data" id = "' + id + '"' + lengthLimit + ' placeholder = "' + placeholder + '" value="' + value + '" disabled/>' +
+        '</p>' +
+        '</div>';
+    };
+
+    var nodeForm = function (nodes) {
+      var node = _.sortBy(nodes, function (node) {
+        return [node.nodeNumber, node.type];
+      })[0];
+      return _.template('' +
+        '<header>' +
+        formCommon.captionTitle('Solmun tiedot:') +
+        '</header>' +
+
+        '<div class="wrapper read-only">' +
+        ' <div class="form form-horizontal form-dark">' +
+        '   <div>' +
+        staticField('Solmunumero:', node.nodeNumber) +
+        inputField('*Solumn nimi:', 'name', '', node.name, 32) +
+        inputField('*Solmutyyppi:', 'type', '', getNodeType(node.type).description) +
+        inputField('*Alkupvm    :', 'date', '', node.startDate) +
```

Comment:
Do these spaces really look good in form: "*Alkupvm    :"? I think this should be just "*Alkupvm".